widget/info.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "gridSummary",
+    "title": "Grid Summary",
```

Comment:
we might need to change the title/name to something like Summary Grid

Reply:
name will be same as per the widget repo 
title we can change